authentication/understanding-identity-provider.adoc

```diff
@@ -72,6 +72,12 @@ link:http://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth[Authoriza
 Once an identity provider has been defined, you can
 xref:../authentication/using-rbac.adoc#authorization-overview_using-rbac[use RBAC to define and apply permissions].
 
+[NOTE]
+====
+When configuring multiple identity providers, some of them, such as `Request header` and `HTPasswd` can not be
```

Comment:
Maybe focus on the the `RequestHeader` IdP being hard to combine with some other CLI-enabled providers and give a brief explanation as to why that is (redirects and HTTP challenges don't mix)

Reply:
Clarified and moved to the Request Header IdP section.